CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules" ${CMAKE_MODULE_
 
 option(BUILDING_TESTS "Build unit tests." ON)
 
-project(vdb_mapping CXX)
+project(vdb_mapping CXX C)
```

Comment:
If I have project(vdb_mapping) all by itself it builds, but I get the MPI::C error with just CXX there, could look at verbose output to see what is different

Reply:
According to the cmake doc having just project(vdb_mapping) is equal do having cxx and c in there 
``` By default C and CXX are enabled if no language options are given. Specify language NONE, or use the LANGUAGES keyword and list no languages, to skip enabling any languages. ```
So I would say we keep it in there with cxx and c since its more explicit.